resources/views/layouts/app.blade.php

```diff
@@ -29,9 +29,11 @@
                 <a class="dropdown-item" href="/admin/users">Users</a>
                 <a class="dropdown-item" href="/admin/events">Events</a>
                 <a class="dropdown-item" href="/admin/faset">Faset</a>
+                <a class="dropdown-item" href="/admin/dues/pending">Accept Dues</a>
```

Comment:
This navigation entry name could potentially be confusing. I get why you named it this, but let's discuss potential alternatives.

Reply:
I'm mulling over a whole reorg of the Navbar. Let's have a separate conversation.